### PR TITLE
Tidy up to README.md

### PR DESCRIPTION
**Pull request by GrantBartlett**
A quick and small tidy up to the readme, including a note about requirement of mix files to build.

### CODE REVIEW

**Comment by Phrohdoh** [comment]
This is incorrect. The mix files need to be in the OpenRA content directory for RA2.

File: readme.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ Directory tree:
...
-    2) build OpenRA (in the `openra` directory)
+1. `git clone --recursive https://github.com/Phrohdoh/oramod-ra2.git`
+2.  Build OpenRA (in the `openra` directory)
+3.  Copy the ra2 directory into openra/mods/ (as openra/mods/ra2/)
+4.  Launch OpenRA and select the ra2 mod
 
-    3) copy the ra2 directory into openra/mods/ (as openra/mods/ra2/)
-
-    4) launch OpenRA and select the ra2 mod
+Note: `theme.mix`, `language.mix` and `ra2.mix` files are required in the OpenRA content directory for RA2.
```
Comment (Phrohdoh) [inline readme.md]: Include the Oxford comma here please. `language.mix, and ra2.mix`

**Comment by Phrohdoh** [comment]
Once you add the comma please squash the commits then this can be merged.

**Comment by GrantBartlett** [comment]
Updated

**Comment by Phrohdoh** [comment]
Thanks!

**Pull request merged**